{
  IOException e=new ConnectException("failed");
  IOException wrapped=verifyExceptionClass(e,ConnectException.class);
  assertInException(wrapped,"failed");
  assertWikified(wrapped);
  assertInException(wrapped,"localhost");
  assertRemoteDetailsIncluded(wrapped);
  assertInException(wrapped,"/ConnectionRefused");
}
